test-case-reporting/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "amp-test-case-reporting",
+  "author": "The AMP HTML Authors",
+  "version": "0.0.1",
+  "private": true,
+  "repository": "http://github.com/ampproject/amp-github-apps",
+  "description": "An app that tracks the pass/fail status of tests run in Travis builds",
+  "engines": {
+    "node": "^12.13.1"
+  },
+  "scripts": {
+    "lint": "eslint **/*.ts",
+    "fix": "eslint **/*.ts --fix",
+    "lint-conflicts": "eslint --print-config app.ts | eslint-config-prettier-check",
+    "build": "tsc",
+    "build:watch": "tsc -w --p tsconfig.json",
+    "start": "node dist/index.js",
+    "dev": "nodemon",
+    "test": "jest",
+    "test:watch": "jest --watch --notify --notifyMode=change"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "dotenv": "8.2.0",
+    "express": "^4.17.1"
```

Comment:
Remove ^ when setting versions in the file because we purposely update packages after testing.

Reply:
Done, I think.

Reply:
Good catch!

Reply:
Context: The caret means "at least", whereas we want exact versions. `npm install --save-exact` will do this for you